stable_baselines3/common/on_policy_algorithm.py

```diff
@@ -92,6 +92,7 @@ def __init__(
             use_sde=use_sde,
             sde_sample_freq=sde_sample_freq,
             support_multi_env=True,
+            monitor_wrapper=monitor_wrapper,
```

Comment:
nice catch =) (we need to mention that in the changelog)

Reply:
How should we mention it ?

Reply:
something like "Fixed monitor_wrapper argument that was not passed to the parent class`